src/FieldContext.tsx

```diff
@@ -99,17 +104,21 @@ function useContextProt(name: string) {
  *  </>
  * )
  */
-export function useTsController<FieldType extends any>() {
+export function useTsController<FieldType extends any>({
+  name,
+}: {
+  name?: string;
```

Comment:
this lets us override the name for tsController so components can still useTsController even if they want to customize the path. i'll comment below where i'm using this

Reply:
maybe we could automatically derive the prefix here as well? just like we do in FormFragment? i think that might be safer